tests/gdimagecopymerge/Makemodule.am

```diff
@@ -0,0 +1,7 @@
+if HAVE_LIBPNG
+	libgd_test_programs += \
```

Comment:
don't indent the bodies in Makemodule.am files under tests/

Reply:
do you mind telling me why

Reply:
we haven't been doing it thus far and i don't think we need to start, and it makes it somewhat easier to diff cmake & autotool files